clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -833,7 +833,6 @@ describe("Consent banner", () => {
 
       it("does not render banner", () => {
         cy.get("div#fides-banner").should("not.exist");
-        cy.contains("button", "Accept Test").should("not.exist");
```

Comment:
Sounds like we need to pick one approach:

- Use `cy.wait` for now
- Unsure how this would work, but the test env could use a shorter delay for the banner, and pass that in somehow as a prop

Reply:
yeah, I was thinking the latter, but also hadn't thought through how it'd work yet

Reply:
we'd probably have to create a new `npm run` script for just test env, and specifically set a var there to signal we're on test. This will require some restructuring and time. Could you create a follow-up, and for now update tests to use a small cy.wait?

Reply:
ahh this makes sense. So for legacy consent, where overlay is disabled, we can assert `cy.get("div#fides-banner").should("not.exist");` because it actually shouldn't be rendered in the DOM, but for notice-based consent, se should use `cy.get("div#fides-banner").should("not.be.visible");`, because it will be rendered in the DOM, just hidden.